modelscope/models/cv/gaussian_splatting_4D/gaussian_splatting_4D.py

```diff
@@ -0,0 +1,135 @@
+import math
+import os
+import os.path as osp
+import sys
+from typing import Any, Dict
+
+import einops
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.transforms as TF
+from PIL import Image
+from shotdetect_scenedetect_lgss import shot_detector
+from tqdm import tqdm
+
+from modelscope.metainfo import Models
+from modelscope.models.base.base_torch_model import TorchModel
+from modelscope.models.builder import MODELS
+from modelscope.utils.config import Config
+from modelscope.utils.constant import ModelFile, Tasks
+from modelscope.utils.logger import get_logger
+
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+import imageio
+import numpy as np
+import torch
+import os
+import cv2
+from tqdm import tqdm
+from os import makedirs
+from .gaussian_renderer.render import render
+from .scene.scene import Scene
+
+import torchvision
+from .utils.general_utils import safe_state
+from argparse import ArgumentParser
+from .arguments.params import ModelParams, PipelineParams, get_combined_args, ModelHiddenParams
+# from .gaussian_renderer import GaussianModel
+from .scene.gaussian_model import GaussianModel
+
+from time import time
+
+import mmcv
+from .utils.params_utils import merge_hparams
+logger = get_logger()
+
+
+@MODELS.register_module(
+    Tasks.gaussian_splatting_4D, module_name=Models.gaussian_splatting_4D)
+
+class GaussianSplatting4D(TorchModel):

```

Comment:
add docstr for model and paper link if it has

and why does this model not have forward function